mslib/msui/mpl_pathinteractor.py

```diff
@@ -336,7 +336,7 @@ class PathPlotter:
     def __init__(self, ax, mplpath=None,
                  facecolor='blue', edgecolor='yellow',
                  linecolor='blue', markerfacecolor='red',
```

Comment:
when you want to keep the stored values you need to use them here. Here you initialize the new line with a thickness of 2.

Reply:
Have a look on redraw_path

Reply:
and set_settings in mpl_qtwidget there we use the other options for initializing them on e.g. a second topview.